Makefile

```diff
@@ -19,10 +19,12 @@ SHELL := /usr/bin/env bash
 ifeq ($(RELEASE),)
   CARGO_BUILD_ARGS :=
   XCODE_CONFIG := Debug
+  GRADLE_BUILD_TYPE := Debug
```

Comment:
Looks like `XCODE_CONFIG` and `GRADLE_BUILD_TYPE` are always equivalent. Could it be reasonable to unify them into some kind of `RELEASE_MODE_CAPITALIZED`?

Reply:
Eh, I had weighed that before I pushed the PR, but eventually decided against it because I felt it was better to leave different build systems' options isolated. I don't have a strong opinion on it though.